python/hidet/drivers/build_graph.py

```diff
@@ -145,6 +145,11 @@ def get_graph_meta_data(graph: FlowGraph, num_kernels, space: int) -> GraphMetaD
         lines.append(str(node.task))
     lines.append(str(graph))
```

Comment:
Will this line produce different string when the topological order is different?

Reply:
Yes, I think anything that relies on `graph.nodes` will cause this.